format/midi/.gitignore

```diff
@@ -0,0 +1,5 @@
+.DS_Store
+workdir
+Makefile
```

Comment:
Add the Makefile? i usually keep scripts etc and a README.md in `testdata` to document how the test files were generated

Reply:
It's just some scrappy targets for dev and debug - nothing of value. I'll move it to the _workdir_ where it's invisible.

Reply:
👍 great! is good if someone (including oneself 😬) in the future needs to regenerate/update the test files

Reply:
Good point! Have added a cleaned up Makefile to _formats/midi/testdata_.